server/game/cards/07-WotW/KaitoNobukai.js

```diff
@@ -0,0 +1,25 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes } = require('../../Constants');
+
+class KaitoNobukai extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Bow each participating characters',
+            cost: ability.costs.sacrificeSelf(),
+            condition: context => context.source.isParticipating(),
+            gameAction: [
+                ability.actions.bow(() => ({
+                    target: this.game.findAnyCardsInPlay(card => card.getType() === CardTypes.Character && card.isParticipating())
+                })),
+                ability.actions.cardLastingEffect(() => ({
+                    target: this.game.findAnyCardsInPlay(card => card.getType() === CardTypes.Character),
+                    effect: ability.effects.cardCannot('moveToConflict')
+                }))]
```

Comment:
It is now preferred that you use `AbilityDsl.actions.multiple([action1, action2])` instead of passing an array directly to `gameAction`.

(All of this needs updating in the guide for implementing cards)

Reply:
Ah thanks for the heads up, will fix it tonight.